src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/UsersController.cs

```diff
@@ -93,12 +93,44 @@ public async Task<ActionResult<GetUser>> GetUserById(string userId)
             {
                 Name = user.DisplayName,
                 PhoneNumber = user.PhoneNumber,
-                EmailAddress =  user.Email,
+                EmailAddress = user.Email,
                 Disabled = user.Disabled,
                 PrimaryOrganisationId = user.PrimaryOrganisationId
             };
 
             return Ok(getUser);
         }
+
+        [Route("api/v1/users/{userid}/enable")]
+        [HttpPost]
```

Comment:
I personally think that, due to the nature of the request, this should be a PUT endpoint, rather than POST, but I'm open to discuss this. 
On this note, if we stick with POST, should we perhaps return something other than 204? 

Opinions? @RuthBerry96 @sam-roberts-bjss @CallumBradburyBJSS  @bjss-chris

Reply:
I think I'd be more inclined to go with PATCH here as it's a partial update but will be interesting to see what others think.

Reply:
I think it's a PATCH too!

Reply:
Due to the discussion on Friday I am going to leave it as POST as this is what the JSON contract stated